src/components/WalletSwitchButton.tsx

```diff
@@ -34,20 +33,19 @@ interface WalletSwitchButtonProps {
 
 const WalletSwitchButton = ({ style }: WalletSwitchButtonProps) => {
   const activeWalletName = useAppSelector((s) => s.activeWallet.name)
-  const { ref: walletSwitchModalRef, open: openWalletSwitchModal, close: closeWalletSwitchModal } = useModalize()
+
+  const [modalOpen, setModalOpen] = useState(false)
```

Comment:
Yeah I hesitated - didn't know which convention to follow and went for the shortest one. I prefer `isModalOpen` for the state, but I also prefer `setModalOpen` (rather than `setIsModalOpen`) for the function 😝 Anyway, don't care, I'm fine with both.